spec/factories/merchant.rb

```diff
@@ -1,5 +1,6 @@
 FactoryBot.define do
   factory :merchant do
     name { "Sprouts" }
+    status { "Disabled" }
```

Comment:
status is disabled by default when the merchant is created, so this line isn't needed

Reply:
nm! I confused this w items